### PR TITLE
Fix metadata and index numbering

### PR DESCRIPTION
**Pull request by cdinnison**
• Use title instead of ID for buffer and mailchimp
• Format index to start at 1
• Some mailchimp fixes

### CODE REVIEW

File: whitelabel_automator.egg-info/PKG-INFO [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Metadata-Version: 1.1
```
Comment (4ndygu) [inline whitelabel_automator.egg-info/PKG-INFO:1]: the `egg-info` is autogenerated - could we cut this directory out? i can include it in the gitignore for the future.

Comment (4ndygu) [reply whitelabel_automator.egg-info/PKG-INFO:1]: [same thing with removing the dist directory]. otherwise LGTM!

Comment (cdinnison) [reply whitelabel_automator.egg-info/PKG-INFO:1]: yeah we can add it to the gitignore! works for me

Comment (4ndygu) [reply whitelabel_automator.egg-info/PKG-INFO:1]: sgtm! deleted theses file off master and added to the .gitignore.

**Pull request merged**